sdk/communication/Azure.Communication.CallAutomation/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.1.0 (Unreleased)
```

Comment:
this is set as "unreleased", are we planning to update this later?

Reply:
yes, some more changes are expected for ga2